src/components/FloatingHelper/HelperContent/HelperContent.spec.tsx

```diff
@@ -42,8 +43,19 @@ describe('HelperContent', () => {
       const actionText = 'Click Me!'
       const driver = createDriver(<HelperContent actionText={actionText} />);
       expect(driver.hasActionButton()).toBeTruthy();
-      expect(driver.getActionButtonContent()).toBe(actionText);
+      expect(driver.getActionButtonDriver().getTextContent()).toBe(actionText);
+    });
+
+    it('should have white action-button skin by default', () => {
```

Comment:
you also check the default priority, so describe it as well

Reply:
fixed